net-proxy/byedpi/byedpi-0.12.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit systemd toolchain-funcs
+
+DESCRIPTION="Bypass DPI SOCKS proxy"
+HOMEPAGE="https://github.com/hufrea/byedpi/"
+SRC_URI="https://github.com/hufrea/byedpi/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+src_compile() {
+	tc-export CC
```

Comment:
Also `export CFLAGS` since the PMS doesn't guarantee that and we need the `CFLAGS` as environment variables for make to pick them up.